component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/EmailOTPAuthenticator.java

```diff
@@ -2578,4 +2588,28 @@ private boolean isOTPMismatched(AuthenticationContext context) {
 
         return Boolean.parseBoolean(String.valueOf(context.getProperty(EmailOTPAuthenticatorConstants.CODE_MISMATCH)));
     }
+
+    private String getTenantDomainFromAuthenticatedUser(AuthenticatedUser authenticatedUser) {
```

Comment:
Looks like these are reusable methods(used in https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/113 too). Shall we move them to the `IdentityTenantUtils` ?